simulator/func_sim/alu.h

```diff
@@ -249,26 +249,26 @@ struct ALU
     // Bit manipulations
     template<typename I> static void sbinv( I* instr) { instr->v_dst[0] = instr->v_src[0] ^ ( lsb_set<typename I::RegisterUInt>() << shamt_v_src2<typename I::RegisterUInt>( instr)); }
     template<typename I> static void sbext( I* instr) { instr->v_dst[0] = 1U & ( instr->v_src[0] >> shamt_v_src2<typename I::RegisterUInt>( instr)); }
-    
-    template<typename I> static 
+
+    template<typename I> static
     void max( I* instr)
-    { 
+    {
         instr->v_dst[0] = instr->v_src[ge( instr) ? 0 : 1];
     }
-    
-    template<typename I> static 
-    void maxu( I* instr)  
+
+    template<typename I> static
+    void maxu( I* instr)
```

Comment:
Please, put trailing whitespaces back. They may create merge conflicts, which may be confusing for other participants. We'll fix whitespaces once everyone submits tasks.

Reply:
Ok